src/migration/prod/italynorth.tf

```diff
@@ -0,0 +1,53 @@
+resource "azurerm_resource_group" "migration" {
+  name     = "${local.project_itn}-${local.environment.domain}-${local.environment.app_name}-rg-${local.environment.instance_number}"
+  location = "italynorth"
+
+  tags = local.tags
+}
+
+# Create Azure Data Factory instances
+# Enables system-assigned managed identity for secure access to resources
+resource "azurerm_data_factory" "this" {
+  name                = "${local.project_itn}-${local.environment.domain}-${local.environment.app_name}-adf-${local.environment.instance_number}"
+  location            = "italynorth"
+  resource_group_name = azurerm_resource_group.migration.name
+
```

Comment:
Now is allowed public access as default, maybe you can add
`public_network_enabled = false`

Reply:
Yes, I need to understand whether it needs some other resources to work privately